root.html

```diff
@@ -11,29 +11,9 @@
 
     <title>Mattermost</title>
```

Comment:
The new webpack plugin manages injecting most of these. The icons under `Android add to homescreen` don't need to be present as `<link>` tags, since the `manifest.json` already covers this as per https://developers.google.com/web/fundamentals/web-app-manifest/.